hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilder.java

```diff
@@ -128,7 +128,13 @@ public final class SingleTransactionRecordBuilder
     public final List<AbstractMap.SimpleEntry<ContractBytecode, Boolean>> contractBytecodes = new ArrayList<>();
 
     public SingleTransactionRecordBuilder(@NonNull final Instant consensusNow) {
+        this(requireNonNull(consensusNow, "consensusNow must not be null"), null);
+    }
+
+    public SingleTransactionRecordBuilder(
+            @NonNull final Instant consensusNow, @Nullable final Instant parentConsensusTimestamp) {
```

Comment:
I'd prefer if `parentConsensusTimestamp` is `@NonNull`

Reply:
Updated.